src/main/java/hu/oe/nik/szfmv/automatedcar/systemcomponents/Shitfer.java

```diff
@@ -0,0 +1,42 @@
+package hu.oe.nik.szfmv.automatedcar.systemcomponents;
+
+import hu.oe.nik.szfmv.automatedcar.virtualfunctionbus.VirtualFunctionBus;
+
+public class Shitfer {
+    public enum ShiftPos {P, R, N, D}
```

Comment:
There is an `enum` for this already on `master` for the `PowerTrain`. I'm fine with this, just saying that it is redundant, possibly should use only one of them in the future.